### DISCUSSION TITLE
Connect with backend

### DISCUSSION THREAD

**Discussion by 4irgl0w** [neuefische/questions#633] [Web] [answered]
Hey I've started to connect my capstone project to mondoDB but I've got the problem that my data is sometimes undefined and sometimes it works. I've created a test page and used my data to created for example a List with {game.title} elements and it worked after I build a isLoading if case. Now I want to do the same on my other pages but data is here always undefined. I also created an if case with isLoading but it doesn't work and I have no idea why. Here is the link to my branch:
https://github.com/Airglow98/Capstone-Project/tree/us9

**Comment by julialoeschel** [comment, marked as answer]
In the _app.js, you need to provide the fetcher
`const fetcher = (...args) => fetch(...args).then((res) => res.json());`

and use it for your SWR call.
`  const { data, isLoading } = useSWR("api/games", fetcher);`

In the pages, the fetcher will be provided by SWRConfig 👍 
this should lead to recieing data from api/games and work with it.
